enterprise/frontend/src/embedding-sdk/store/selectors.ts

```diff
@@ -29,3 +29,6 @@ export const getMetabaseInstanceUrl = (state: SdkStoreState) =>
 
 export const getFetchRefreshTokenFn = (state: SdkStoreState) =>
   state.sdk.fetchRefreshTokenFn;
+
+export const getIsSdkIframeEmbedAuth = (state: SdkStoreState) =>
```

Comment:
As mentioned in `enterprise/frontend/src/embedding-sdk/store/reducer.ts`, if we move this data to an environment variable, we probably could delete this file as well.